src/Microsoft.Health.Fhir.SqlServer/Features/Schema/Migrations/38.diff.sql

```diff
@@ -0,0 +1,26 @@
+﻿/***************************************************************************
+    Stored procedures for get NonCompleted Job Count Of SpecificQueueType
+****************************************************************************/
+--
+--  STORED PROCEDURE
+--      GetNonCompletedJobCountOfSpecificQueueType
```

Comment:
Can we add some information that this SP is used in health-paas for import functionality?